### DISCUSSION TITLE
Error: could not find function "%>%"

### DISCUSSION THREAD

**Discussion by algoacademy** [teamalgoritma/community#7] [Q&A] [answered]
Halo, saya mendapat error ketika menggunakan function. 
Code saya seperti di bawah:
```
iris %>% 
  filter(Species != "Setosa")
```
Error saya:
```
Error in iris %>% fliter(Species != "Setosa") : 
  could not find function "%>%"
```

**Comment by inytss** [comment, marked as answer]
Error tersebut terjadi dikarenakan fungsi "%>%" terdapat pada package dplyr, sehingga perlu di load terlebih dahulu package dplyr menggunakan perintah
`library(dplyr)`

**Reply by algoacademy** [reply]
Saya tulis `library(dplyr)` nya dimana ya, Mbak?

**Reply by NabiilahArdini** [reply]
Bisa ditulis di dalam chunk pada R Markdown Bapak. Diletakan sebelum code Bapak sebelumnya.

**Comment by onlyphantom** [comment]
@inytss 's solution is the correct one. You can alternatively make an explicit call to `dplyr` using the syntax `dplyr::filter(Species != "Setosa")`.

```
iris %>% 
  dplyr::filter(Species != "Setosa")
```

PS: In the code you pasted you may have misspelled `filter`. Correction: `fliter` -> `filter` :)
